code/game/objects/items.dm

```diff
@@ -317,6 +317,8 @@
 		. += desc
 	if(desc_lore)
 		. += SPAN_NOTICE("This has an <a href='byond://?src=\ref[src];desc_lore=1'>extended lore description</a>.")
+	if(HAS_TRAIT(src, TRAIT_ITEM_DISSOLVING))
+		. += SPAN_WARNING("It is currently dissolving into bits!")
```

Comment:
Turn this into an element instead of a trait so that you don't need to add this to the base item examine proc